service/history/task/interface.go

```diff
@@ -51,6 +51,12 @@ type (
 		GetAttempt() int
 	}
 
+	CrossClusterTask interface {
+		Task
+		IsReadyForPoll() bool
+		Update(interface{}) error
```

Comment:
Will this `interface{}` type be changed to something else, or this is the final state?

Reply:
Added a TODO. We should update it once I have the idl changes in